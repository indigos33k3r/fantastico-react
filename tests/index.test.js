import React from 'react'
import { shallow } from 'enzyme'
import { create } from 'react-test-renderer'
import Main from '../app/components/main'

describe('main component test', () => {
  it('matches the snapshot', () => {
    const component = <Main />
    const tree = create(component).toJSON()
    expect(tree).toMatchSnapshot()
  })
  it('renders the component', () => {
    const wrapper = shallow(<Main />)
    const assertion = wrapper.find('.fantastico').length
    expect(assertion).toBe(1)
  })
})
