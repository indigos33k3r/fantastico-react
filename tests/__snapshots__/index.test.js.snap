// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index.js test matches the snapshot 1`] = `
<div
  className="fantastico"
>
  <h1>
    ðŸ§ž FANTASTICO! is operating
    <br />
    with React!
  </h1>
</div>
`;
